sdk/spring/azure-spring-boot-starter-active-directory/README.md

```diff
@@ -186,9 +82,34 @@ public class AADOAuth2LoginConfigSample extends AADWebSecurityConfigurerAdapter
 }
 ```
 
-### [Web APP] Configure scopes of multiple resources
-By default, `azure-spring-boot-starter-active-directory` configures scopes of `openid`, `profile` and `https://graph.microsoft.com/user.read` to implement OpenID Connect protocol and access of membership information of logging in users.
+### Web application with function of visiting resource servers
+
+#### System diagram
+![Web Application Access Resources](resource/add-based-web-application-access-resources.png)
```

Comment:
![image](https://user-images.githubusercontent.com/13167207/106108245-f0e34100-6182-11eb-9267-57b0d4886fae.png)
It should be `Web application` and `Resource Server`.